setup.cfg

```
@@ -23,6 +23,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

```suggestion
    Programming Language :: Python :: 3.10
    Programming Language :: Python :: 3.11    
```
I tested python 3.11 locally and it works fine. 

There is no appveyor suppor for 3.11 at this moment. Adding 3.11 support to travis is possible, but not on the xenial distribution (see https://docs.travis-ci.com/user/languages/python/#python-versions)